src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineRunCommandTests.ps1

```diff
@@ -113,14 +113,86 @@ function Test-VirtualMachineSetRunCommand
         $st = $job | Receive-Job;
         Assert-NotNull $st.Value;
 
-        # Remove All VMs
-        Get-AzureRmVM -ResourceGroupName $rgname | Remove-AzureRmVM -Force;
-        $vms = Get-AzureRmVM -ResourceGroupName $rgname;
-        Assert-AreEqual $vms $null; 
+        $vm = Get-AzureRmVM -ResourceGroupName $rgname -Name $vmname;
+        $result = Invoke-AzureRmVMRunCommand -ResourceId $vm.Id -CommandId $commandId -ScriptPath $path -Parameter $param;
+        Assert-NotNull $result.Value;
+
+        $result = Get-AzureRmVM -ResourceGroupName $rgname -Name $vmname | Invoke-AzureRmVMRunCommand -CommandId $commandId -ScriptPath $path -Parameter $param;
+        Assert-NotNull $result.Value;
+
+        $vm = Get-AzureRmVM -ResourceGroupName $rgname -Name $vmname;
     }
     finally
     {
         # Cleanup
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Test Virtual Machine Scale Set VM Run Command Set
+#>
+function Test-VirtualMachineScaleSetVMRunCommand
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName
+
+    try
+    {
+        # Common
+        $loc = Get-Location "Microsoft.Compute" "virtualMachines";
+        New-AzureRMResourceGroup -Name $rgname -Location $loc -Force;
+
+        # NRP
+        $subnet = New-AzureRMVirtualNetworkSubnetConfig -Name ('subnet' + $rgname) -AddressPrefix "10.0.0.0/24";
+        $vnet = New-AzureRMVirtualNetwork -Force -Name ('vnet' + $rgname) -ResourceGroupName $rgname -Location $loc -AddressPrefix "10.0.0.0/16" -Subnet $subnet;
+        $vnet = Get-AzureRMVirtualNetwork -Name ('vnet' + $rgname) -ResourceGroupName $rgname;
+        $subnetId = $vnet.Subnets[0].Id;
+
+        # New VMSS Parameters
+        $vmssName = 'vmss' + $rgname;
+
+        $adminUsername = 'Foo12';
+        $adminPassword = $PLACEHOLDER;
```

Comment:
use the [GetPasswordForVM](https://github.com/Azure/azure-powershell/pull/7190/files#diff-c3a91e9dd099c92910feae5283a65883R28) function instead